### ISSUE TITLE
[question] Still limited to 200 connections also with Aedes

### ISSUE THREAD

**Issue by Pakkes** [moscajs/aedes#425]
Hi.
By following the suggestion from @robertsLando I moved to Aedes. Which is a great module!
However I'm still not able to get more than 200 connections on the server. Simply the server stops allowing further connections once reached 200.
Any suggestion on where I could look for a possible setting limiting the #of concurrent connections to the server? I've tried to run the server on a Windows 10 machine and on a  Windows server 2012 machine.
Really many thanks for the help. Regards

**Comment by robertsLando** [comment]
I can't reproduce your problem on my machine. You should provide some more informations about your server configuration and provide a working example

**Comment by gnought** [comment]
@Pakkes we do have a test on concurrent connection https://github.com/moscajs/aedes/blob/fc45bc6cc9bbf01443e6670f2c593dcd280bf20b/test/not-blocking.js#L9
We have tried 500 in Linux and macOS, and all goes fine. You could try to raise the number of connection in the test and see it works in your env.

**Comment by Pakkes** [comment]
Hi. Finally I realized why I got that limitation!!!! I've tried concurrent connections by using WebSockets via browsers (Chrome, Firefox) and it seems that, at least on Windows, there is a 200/203 maximum allowed connections from browser (regardless of the # of browser windows you open). By spanning several browsers on different computers I can get thousands of WebSocket connections. Also, by using the MQTT bench tool [github.com/takanorig/mqtt-bench](url) I can get dozens of thousands of mqtt connections. So, it was a problem of mine.
Many thanks for suggesting me to move to Aedes, I'm happy to use it, it seems really stable and with great performance.
